web/src/components/troubleshoot/SupportBundleRow.tsx

```diff
@@ -165,11 +180,12 @@ class SupportBundleRow extends React.Component {
     });
   };
 
-  moveBar(progressData) {
+  moveBar(progressData: SupportBundleProgress) {
     const elem = document.getElementById("supportBundleStatusBar");
-    const calcPercent =
-      (progressData.collectorsCompleted / progressData.collectorCount) * 100;
-    percentage = calcPercent > 98 ? 98 : calcPercent.toFixed();
+    const calcPercent = Math.round(
+      (progressData.collectorsCompleted / progressData.collectorCount) * 100
+    );
+    percentage = calcPercent > 98 ? 98 : calcPercent;
```

Comment:
- removed `.toFixed()` which is a number -> string conversion. this allows us to keep `percentage` as a number 
- replaced it with `Math.round` which always returns an integer

Reply:
perfect